pkg/util/kubernetes/apiserver/diagnosis.go

```diff
@@ -19,12 +25,44 @@ func init() {
 
 // diagnose the API server availability
 func diagnose() error {
-	isConnectVerbose = true
 	c, err := GetAPIClient()
-	isConnectVerbose = false
 	if err != nil {
 		return err
 	}
 	log.Infof("Detecting OpenShift APIs: %s available", c.DetectOpenShiftAPILevel())
+
+	resourcesNamespace := common.GetResourcesNamespace()
```

Comment:
@clamoriniere Added listing permissions in current or resources namespace. It does not appear to be possible to get cluster wide permissions, only to check for a known resource.